src/core/public/chrome/chrome_service.tsx

```diff
@@ -496,3 +533,25 @@ export interface InternalChromeStart extends ChromeStart {
    */
   getHeaderComponent(): JSX.Element;
 }
+
+export enum GROUP_TYPE {
+  SYSTEM_GROUP = 'SYSTEM_GROUP',
+  USE_CASE_GROUP = 'USE_CASE_GROUP',
+}
```

Comment:
```suggestion
export enum NavGroupType {
  SYSTEM = 'system',
}
```
... and make `type` optional below.

Note that the `enum` name is in Pascal-case, the name of the the actual constant is uppercase, with no suffix tying it to the `enum` name, and the value is lowercase.

Reply:
Thanks.